samples/generated/v2/snippet_metadata_google.cloud.bigquery.v2.json

```diff
@@ -1660,4 +1660,4 @@
             }
         }
     ]
-}
\ No newline at end of file
+}
```

Comment:
I do not know why it does not have this closing curly brace upstream - I feel like it's not valid without it. This file will get regenerated and is not integral to this PR so if you prefer I remove this curly brace for consistency I can get rid of it

Reply:
I think is fine to leave it here.